.github/workflows/build.yml

```diff
@@ -0,0 +1,219 @@
+          print('::set-env name=ARTIFACT_NAME::xqemu-ubuntu-release')
+    - name: Clone Tree
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Install Dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+          libsdl2-dev \
+          libepoxy-dev \
+          libpixman-1-dev \
+          ccache
+    - name: Initialize Compiler Cache
+      id: cache
+      uses: actions/cache@v1
+      with:
+        path: /tmp/xqemu-ccache
+        key: cache-${{ runner.os }}-${{ matrix.configuration }}-${{ github.sha }}
+        restore-keys: cache-${{ runner.os }}-${{ matrix.configuration }}-
+    - name: Compile
+      run: |
+        export CCACHE_DIR=/tmp/xqemu-ccache
+        export CCACHE_MAXSIZE=250M
+        export PATH="/usr/lib/ccache:$PATH"
+        ./build.sh ${{ env.BUILD_PARAM }}
+        echo -e "\nCompiler Cache Stats:"
+        ccache -s -c
+
+  macOS:
+    runs-on: macOS-latest
+    needs: Init
+    strategy:
+      matrix:
+        configuration: ["Debug", "Release"]
+    steps:
+    - name: Define Build Options
+      shell: python
+      run: |
+        if "${{ matrix.configuration }}" == "Debug":
+          print('Configuring for Debug')
+          print('::set-env name=BUILD_PARAM::')
+          print('::set-env name=ARTIFACT_NAME::xqemu-macos-debug')
+        else:
+          print('Configuring for Release')
+          print('::set-env name=BUILD_PARAM::--release')
+          print('::set-env name=ARTIFACT_NAME::xqemu-macos-release')
+    - name: Clone Tree
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Install Dependencies
+      run: |
+        brew update
+        brew unlink python@2
+        brew install \
+          glib \
+          pixman \
+          sdl2 \
+          libepoxy \
+          coreutils \
+          pkg-config \
+          ccache
+    - name: Initialize Compiler Cache
+      id: cache
+      uses: actions/cache@v1
+      with:
+        path: /tmp/xqemu-ccache
+        key: cache-${{ runner.os }}-${{ matrix.configuration }}-${{ github.sha }}
+        restore-keys: cache-${{ runner.os }}-${{ matrix.configuration }}-
+    - name: Compile
+      run: |
+        export CCACHE_DIR=/tmp/xqemu-ccache
+        export CCACHE_MAXSIZE=250M
+        export PATH="/usr/local/opt/ccache/libexec:$PATH"
+        ./build.sh ${{ env.BUILD_PARAM }}
+        echo -e "\nCompiler Cache Stats:"
+        ccache -s -c
+
+  Release:
+    runs-on: ubuntu-latest
+    needs: [Ubuntu, Windows, macOS]
+    env:
+      BUILD_TAG:
+    steps:
+    - name: Download Artifacts
+      uses: actions/download-artifact@v1
+      with:
+        name: dist
+    - name: Get Package Info
+      run: |
+        echo "::set-env name=BUILD_TAG::$(cat dist/tag)"
+    - name: Create Release
+      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+      id: create_release
+      uses: actions/create-release@v1.0.0
```

Comment:
This version number can be shortened to just `v1`.

Reply:
Thanks, copy paste from old documentation